app/controllers/v0/messages_controller.rb

```diff
@@ -26,8 +26,12 @@ def show
     def create
       message = Message.new(message_params)
       raise Common::Exceptions::ValidationErrors, message unless message.valid?
-
-      response = client.post_create_message(message_params)
+      response =
+        if request.content_type == 'multipart/form-data'
```

Comment:
NM, I think I agree with you on this I will make a private helper method.

Basically if params data { type: attachment } then request must be multipart or raise an exception. But I still want to address it as a separate PR. We need to bring all the API endpoints in line with jsonapi expectations and I need to have conversation with F/E on that first.